### PR TITLE
Hiding search input and icon

### PR DESCRIPTION
**Pull request by paulina-pacheco**

### CODE REVIEW

File: public/stylesheets/sections/_labs.sass [hunk 1]
```diff
@@ -35,7 +35,7 @@
     display: inline-flex
 
 .news-search
-
+  display: none !important
```
Comment (echavezNS) [inline public/stylesheets/sections/_labs.sass]: Commonly is a bad practice to use !important, try to use a hierarchy in order to apply this style.

File: views/pages/labs.jade [hunk 2]
```diff
@@ -38,51 +37,55 @@ block content
...
+  div.news.wrapper-12
+    div.small-9.small-centered
+      div.news-top.small-12
+        div.section-title.small-12.medium-6
+          h2 NEWS
+        div.news-search.small-12.medium-6
+          div.news-input-wrapper.small-11
+            input(type="text", placeholder="search")
+          div.news-icon-wrapper.small-1
+            img.search-icon(src="images/search.png")
```
Comment (echavezNS) [inline views/pages/labs.jade]: All images should contain an `alt` attribute, this is good for search engines and better ranking.

File: views/pages/labs.jade [hunk 3]
```diff
@@ -38,51 +37,55 @@ block content
...
+            div.content-title
+              h2 Atlassian<span class="news-title-highlight" style="font-size: 80px">&#43;</span>Nearsoft
+            div.content-text
+              p Además de ser una buena oportunidad para desarrollar add-ons, el haber participado en dos Connect Weeks nos ha permitido estar en contacto con el equipo de Atlassian. Esto a su vez nos ha brindado varias oportunidades interesantes al equipo de Labs y también a Nearsoft.
+            div.content-title
+              h2 Workshop <span class="news-title-highlight">(HMO)</span> - From Idea to MVP
+            div.content-text
+              p El próximo taller, From Idea to MVP impartido en Hermosillo, tratará de cómo describir a fondo una idea usando el Product Canvas, para poder así definir un MVP para validar la propuesta de valor con usuarios potenciales antes de empezar a desarrollar.
+          div.scroll
+            img.scroll-icon(src="images/scroll.png")
```
Comment (echavezNS) [inline views/pages/labs.jade]: `alt` attribute

File: views/pages/labs.jade [hunk 4]
```diff
@@ -38,51 +37,55 @@ block content
...
-              div.content-title
-                h2 Atlassian<span class="news-title-highlight" style="font-size: 80px">+</span>Nearsoft
-              div.content-text
-                p Además de ser una buena oportunidad para desarrollar add-ons, el haber participado en dos Connect Weeks nos ha permitido estar en contacto con el equipo de Atlassian. Esto a su vez nos ha brindado varias oportunidades interesantes al equipo de Labs y también a Nearsoft.
-              div.content-title
-                h2 Workshop <span class="news-title-highlight">(HMO)</span> - From Idea to MVP
-              div.content-text
-                p El próximo taller, From Idea to MVP impartido en Hermosillo, tratará de cómo describir a fondo una idea usando el Product Canvas, para poder así definir un MVP para validar la propuesta de valor con usuarios potenciales antes de empezar a desarrollar.
-            div.scroll
-              img.scroll-icon(src="images/scroll.png")
```
Comment (echavezNS) [inline views/pages/labs.jade]: `alt` attribute.

**Comment by paulina-pacheco** [comment]
I made some changes as you suggested, @echavezNS. Please take a look :)

**Pull request merged**